content/enterprise/install/interactive/installer.mdx

```diff
@@ -159,6 +159,8 @@ Terraform Enterprise uses a standard set of ciphersuites by default. If necessar
 
 (This value must be defined in the OpenSSL [cipher list format](https://www.openssl.org/docs/man1.1.1/man1/ciphers.html)
 
+~> Note: By default enabled ciphers for Nginx are `HIGH:!aNULL:!MD5`.
```

Comment:
I know we're a bit inconsistent about notes (I promise fixes for this are coming!) but I'm adjusting your callout formatting so that it matches the majority of other notes in the Terraform docs right now.  Also a tiny wording nit :-) 

```suggestion
-> Note: The default enabled ciphers for Nginx are `HIGH:!aNULL:!MD5`.
```

Reply:
Thank you, looks great!